h/templates/rss.xml.jinja2

```diff
@@ -0,0 +1,28 @@
+{# Generic template for rendering an RSS feed given a feed dict.
+
+The feed dict should be a logical representation of an RSS feed as a Python
+dict, including a list of dicts for the feed's items. This template will
+render the feed to RSS XML.
+
+-#}
+<?xml version="1.0"?>
+<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
```

Comment:
Yeah, the validator told me to put that in, I thought it was wrong at first too!